_raml/apps/products/products.raml

```diff
@@ -97,7 +97,8 @@ post:
                 - /priceInfo/depositPrice/amount<br>
                 - /priceInfo/taxClass/taxClassId³<br>
                 - /productImage¹<br>
-                - /stocklevel²<br><br>
+                - /stocklevel²<br>
+                - /visible<br><br>
```

Comment:
I took the chance to review all attributes listed here and compared them with our code:
we are missing also:

- deliveryPeriod
- minStocklevel
- manufacturerProductNumber
- productLength
- productWidth
- productHeight

(all of them are normal cases and don't need to be flagged with a small 1 or 2)